tgui/packages/tgui/interfaces/GreyscaleModifyMenu.tsx

```diff
@@ -0,0 +1,117 @@
+import { useBackend } from '../backend';
+import { Box, Button, ColorBox, Flex, Icon, Input, LabeledList, Section, Stack, Table } from '../components';
+import { Window } from '../layouts';
+
+type ColorEntry = {
+  index: Number;
+  value: string;
+}
+
+type SpriteData = {
+  finished: SpriteEntry
+  steps: Array<SpriteEntry>
+}
+
+type SpriteEntry = {
+  layer: string
+  result: string
+}
+
+type GreyscaleMenuData = {
+  colors: Array<ColorEntry>;
+  sprites: SpriteData;
+}
+
+const ColorDisplay = (props, context) => {
+  const { act, data } = useBackend<GreyscaleMenuData>(context);
+  const colors = (data.colors || []);
+  return (
+    <Section title="Colors">
+      <LabeledList>
+        <LabeledList.Item
+          label="Full Color String">
+          <Input
+            value={colors.map(item => item.value).join('')}
+            onChange={(_, value) => act("recolor_from_string", { color_string: value })}
+          />
+        </LabeledList.Item>
+        {colors.map(item => (
+          <LabeledList.Item
+            key={`colorgroup${item.index}${item.value}`}
+            label={`Color Group ${item.index}`}
+            color={item.value}
+          >
+            <ColorBox
+              color={item.value}
+            />
+            {" "}
+            <Button
+              content={<Icon name="palette" />}
+              onClick={() => act("pick_color", { color_index: item.index })}
+            />
+            <Input
+              value={item.value}
+              onChange={(_, value) => act("recolor", { color_index: item.index, new_color: value })}
+            />
+          </LabeledList.Item>
+        ))}
+      </LabeledList>
+    </Section>
+  );
+};
+
+const PreviewDisplay = (props, context) => {
+  const { data } = useBackend<GreyscaleMenuData>(context);
+  return (
+    <Section title="Preview">
+      <Table>
+        <Table.Row header>
+          <Table.Cell><Box textAlign="center" bold>Step Layer</Box></Table.Cell>
+          <Table.Cell><Box textAlign="center" bold>Step Result</Box></Table.Cell>
+        </Table.Row>
```

Comment:
You don't need bold because header row already makes things bold, and i think you can apply textAlign="center" to the whole row as well, it cascades to all the text inside. And there would be no need in the box.

Reply:
re: textAlign, it's still on the cells after realizing it doesn't work on the row